docs/source/extend_kedro/common_use_cases.md

```diff
@@ -31,7 +31,7 @@ def notebook_run(...):
     == ADD YOUR CUSTOM NOTEBOOK COMMAND CODE HERE ==
 ```
 
-To inject additional CLI commands intended to be reused across projects, see [our plugin system](./plugins.md). An example of one such command is the `kedro viz` command introduced by the [Kedro-Viz plugin](https://github.com/kedro-org/kedro-viz). This command is intended to work on every Kedro project and therefore must be a standalone plugin.
+To inject additional CLI commands intended to be reused across projects, see [our plugin system](./plugins.md). An example of one such command is the `kedro viz run` command introduced by the [Kedro-Viz plugin](https://github.com/kedro-org/kedro-viz). This command is intended to work on every Kedro project and hence must be a standalone plugin.
```

Comment:
```suggestion
To inject additional CLI commands intended to be reused across projects, see [our plugin system](./plugins.md). An example of one such command is the `kedro viz run` command introduced by the [Kedro-Viz plugin](https://github.com/kedro-org/kedro-viz). This command is intended to work on every Kedro project and it must be a standalone plugin.
```

Reply:
It's not that clear from reading this -- I'm left thinking "how is a command a standalone plugin"? So I've made an edit to try to clarify slightly but would appreciate some insight from @merelcht or @astrojuanlu or @noklam as to whether this now reads correctly.

Reply:
Hmmmm good point - the original phrasing is not clear to me either.

> This command is intended to work on every Kedro project and hence must be a standalone plugin.

Regardless of @stichbury edit, what does this mean?